### PR TITLE
[RFR] Test case for checks

### PR DESCRIPTION
**Pull request by john-dupuy**
Adding unit tests for the rhos checks

To be merged after #7

### CODE REVIEW

File: check-osp/test_checks.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        pytest.fail("Please clone yamls directory in conf/ to run this test.")
+
+    try:
+        with open("conf/cfme-qe-yamls/complete/cfme_data.yaml", "r") as stream:
+            cfme_data = yaml.safe_load(stream)
+    except IOError:
+        pytest.fail("CFME Data YAML file not found, no provider data available.")
+    else:
+        providers = cfme_data.get("management_systems")
+        rhv_providers = {
```
Comment (kedark3) [inline check-osp/test_checks.py]: Please make it `osp_providers`

File: check-osp/test_checks.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            key: providers[key] for key in list(providers.keys())
+            if providers[key].get("type") == "openstack"
+        }
+        yield rhv_providers.get(random.choice(list(rhv_providers.keys())))
```
Comment (kedark3) [inline check-osp/test_checks.py]: Here as well :)

File: check-osp/test_checks.py [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        auth_url=provider_data.get("auth_url")
+    )
+
+    # now try run the check
+    with pytest.raises(SystemExit) as pytest_wrapped_e:
+        services = None
+        if measurement == "services_status":
+            # since this is a unit test, services don't matter
+            services =  {
+                'vdsmd': 'Active: active (running)',
```
Comment (kedark3) [inline check-osp/test_checks.py]: Please change the service name to something like `nova` and `neutron` 
You may refer yaml for service checks in conf.

File: requirements.txt [hunk 4]
```diff
@@ -1,2 +1,6 @@
-wrapanapi==3.3.1
 paramiko==2.6.0
+PyYAML==5.1.1
+pytest=5.0.1
```
Comment (kedark3) [inline requirements.txt]: This needs to be updated. should be `pytest==5.0.1`

**Review by kedark3 — COMMENTED** [review]
Tests did pass. Please address my comments and we should be good to merge. Also can you please update our README.md to reflect we are now python3? 
Thanks for your work on this :tada:

File: check-osp/test_checks.py (hunk 1 above)
Comment (john-dupuy) [reply check-osp/test_checks.py]: thanks for catching my copy-pasta

**Review by kedark3 — APPROVED** [review]

**Pull request merged**